microsim/opencl/ramp/inspector.py

```diff
@@ -424,8 +424,6 @@ def draw_parameters_window(self):
 
         _, self.params.symptomatic_multiplier = imgui.slider_float(
             "Symptomatic Multiplier", self.params.symptomatic_multiplier, 0.0, 1.0)
-        _, self.params.proportion_asymptomatic = imgui.slider_float(

```

Comment:
worth noting that this is also changed in your other PR, so there may be a merge conflict if that is merged first, or you could just close that other PR without merging and make both changes in this PR

Reply:
Ah woops, think the other one has already been merged - will try and merge with updated master and see if that solves it.